shell/platform/android/BUILD.gn

```diff
@@ -171,56 +173,41 @@ java_library("flutter_shell_java") {
     "io/flutter/view/VsyncWaiter.java",
   ]
 
-  java_files +=
-      [ "$flutter_root/third_party/bsdiff/io/flutter/util/BSDiff.java" ]
+  sources += [ "$flutter_root/third_party/bsdiff/io/flutter/util/BSDiff.java" ]
 
-  deps = [
-    ":android_arch_lifecycle_common",
-    ":android_arch_lifecycle_viewmodel",
-    ":android_support_annotations",
-    ":android_support_compat",
-    ":android_support_fragment",
-    ":android_support_v13",
+  android_support_jars = [
+    "//third_party/android_support/android_support_v13.jar",
+    "//third_party/android_support/android_support_compat.jar",
+    "//third_party/android_support/android_support_annotations.jar",
+    "//third_party/android_support/android_support_fragment.jar",
+    "//third_party/android_support/android_arch_lifecycle_common.jar",
+    "//third_party/android_support/android_arch_lifecycle_viewmodel.jar",
```

Comment:
@dnfield do we know who is using this? The lifecycle_common is needed for the new plugin API proposal, but I don't think lifecycle_viewmodel is required for that purpose. I wasn't sure if someone else was using it for something.

Reply:
These are used by the embedding since the roll to sdk 28

Reply:
I'm not sure exactly which classes consume them - some of them are transitive deps as well.  But if you remove any javac will fail.